kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -294,6 +307,11 @@ define([
                         this.updateView();
                     }.bind(this));
 
+                    $('td:eq(4)', nRow).find('button[data-download]').off('click').on('click', function (e) {
+                        let file = $(e.currentTarget).data('download');
+                        this.downloadFile(Config.url('staging_api_url') + '/download/' + file)
+                    }.bind(this));
```

Comment:
I'll play with that functionality too.
```suggestion
                    };
```